src/components/Navbar/index.tsx

```diff
@@ -1,14 +1,37 @@
-import React from "react";
+import React from 'react';
+import { navbarContainerStyle } from './styles';
+import { Person } from './types';
+import { cx } from '@emotion/css';
 
-const Navbar = () => {
-    return(
-      <div>
-            <a href="">Testing1</a>
-            <a href="">Testing2</a>
-            <a href="">Testing3</a>
-        </div>
-    ); 
-  };
+interface NavbarProps {
+  link: number;
+  link2?: string;
+  person: Person[];
+}
 
+type NavbarTechnotic = (props: NavbarProps) => JSX.Element;
 
-export default Navbar;
\ No newline at end of file
+const Navbar: NavbarTechnotic = ({ link2 }) => {
+  return (
+    <div className={cx(navbarContainerStyle)}>
+      <a href={link2}>Testing1</a>
+      <a href="">Testing2</a>
+      <a href="">Testing3</a>
+    </div>
```

Comment:
```suggestion
    <div className={cx(navbarContainerStyle)}>
      <a href={link2}>Testing1</a>
      <a href="">Rifian</a>
      <a href="">Joce</a>
    </div>
```
kok namanya gini mustinya tuh gini aja